### PR TITLE
Expanded documentation for SAML Federation w/ examples.

### PR DESCRIPTION
**Pull request by scheibling**
### New Features
Adds further documentation on usage of the SAML source for external authentication together with example metadata

### CODE REVIEW

**Review by BeryJu — CHANGES_REQUESTED** [review]
Looks pretty good, some small nitpicks to match the style: https://goauthentik.io/developer-docs/docs/writing-documentation

**Comment by scheibling** [comment]
> Looks pretty good, some small nitpicks to match the style: https://goauthentik.io/developer-docs/docs/writing-documentation

Yep, that was sloppy on my part :-) Should be improved now, I've changed the formatting a bit as well so it's easier to read.

File: website/integrations/sources/saml/index.md [hunk 1]
```diff
@@ -5,3 +5,77 @@ title: SAML
...
+|Allow IDP-Initiated Logins|False|Whether to allow the IDP to log users into authentik without any interaction. Activating this may constitute a security risk since this request is not verified, and could be utilised by an attacker to authenticate a user without interaction on their side.|
+|NameID Policy|Persistent|Depending on what the IDP sends as persistent ID, some IDPs use the username or email address while others will use a random string/hashed value. If the user in authentik receives a random string as a username, try using Email address or Windows|
+|Flow settings|Default|If there are custom flows in your instance for external authentication, change to use them here|
+
+## Adding authentik as a server provider with your IDP
+This will depend heavily on what software you are using for your IDP. On the Metadata tab in the SAML Federation Source you can download the metadata for the service provider, this should enable you to import this into most IDPs. If this does not work, the important parts are:
+
+- Entity ID: Taken from the Issuer/Entity ID field above
+- Return URL/ACS URL: https://authentik.company/source/saml/<source-slug>/acs/
```
Comment (BeryJu) [inline website/integrations/sources/saml/index.md:39]: the placeholder needs to be wrapped in backticks as it's currently interpreted as a tag

Comment (scheibling) [reply website/integrations/sources/saml/index.md]: Yes, done!

**Review by BeryJu — CHANGES_REQUESTED** [review]
Seems like docusaurus doesn't like that either, try this 
```diff

 - Entity ID: Taken from the Issuer/Entity ID field above
-- Return URL/ACS URL: https://authentik.company/source/saml/`<source-slug>`/acs/
+- Return URL/ACS URL: `https://authentik.company/source/saml/<source-slug>/acs/`
 - Certificate: If you have chosen to sign your outgoing requests, use the public side of the certificate that you specified in the settings
```

**Comment by scheibling** [comment]
Should work now, managed to build it locally

**Pull request merged**